data/roles/gecko_t_osx_1100_m1.yaml

```diff
@@ -37,19 +37,21 @@ packages_classes:
   - wget
   - xcode_cmd_line_tools
   - zstandard
+  - telegraf
 
 # Package class parameters
 packages::google_chrome::version: v80.0.3987.106
 packages::mercurial::version: 5.5.2
 packages::nodejs::version: 12.11.1
 packages::python2::version: 2.7.18
 packages::python2_zstandard::version: 0.11.1
-packages::python3::version: 3.7.9
+packages::python3::version: 3.9.6
```

Comment:
I assume this python package has already been uploaded to the s3 package repo.  if not, we will need to do that.

Reply:
:+1: This python package is in the s3 package repo.